### DISCUSSION TITLE
SQL Editor font style

### DISCUSSION THREAD

**Discussion by RuudSchmeitz** [pawelsalawa/sqlitestudio#4116] [Q&A]
A question about the SQL Editor font style .
Despite the fact that the `font style` is `Regular` :

![SQL editor font](https://user-images.githubusercontent.com/69636360/118911017-bcf47d00-b925-11eb-9b3e-8a23bfa02c7f.png)

(some of the) keywords are displayed with `font style` `Bold` :

![SQL editor](https://user-images.githubusercontent.com/69636360/118911055-d4336a80-b925-11eb-8482-5bcd751b9c8e.png)

Is there a way to change this ?

**Comment by pawelsalawa** [comment]
Currently not, but I've reported this #4117 to track it.
